int32 robotId
---
int32[] rows
int32[] cols
bool result
